api/src/main/java/uk/gov/dvsa/mot/persist/VehicleReadDao.java

```diff
@@ -34,12 +37,22 @@
 
     List<Vehicle> getVehicleByFullRegistration(String registration);
 
+    List<Vehicle> getVehicleByDvlaVehicleId(Integer dvlaVehicleId);
+
+    List<DvlaVehicle> getDvlaVehicleByFullRegistration(String registration);
+
+    List<DvlaVehicle> getDvlaVehicleByDvlaVehicleId(Integer dvlaVehicleId);
```

Comment:
`DvlaVehicle` shouldn't be returned from the service, it's a persistence layer object. We should return something that's from `api` package/namespace.